code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -1,197 +1,246 @@
+	armor_penetration = ARMOR_PENETRATION_10X24
+	penetrating = 1
+	sharp = FALSE
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/cl44r
-	name = "rubber bullet"
-	damage = 8
-	agony = 65
-	embed = 0
-	sharp = 0
-
-/obj/item/projectile/bullet/c45
-	damage = damage_45
+/obj/item/projectile/bullet/a556
+	damage = DAMAGE_556
+	armor_penetration = ARMOR_PENETRATION_556
+	penetrating = 2
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/c45p
-	damage = 7
-	agony = 7
-	embed = 0
-	sharp = 0
+/obj/item/projectile/bullet/a556/practice
+	damage = DAMAGE_556 * PRACTICE_MULTIPLIER
+	agony = DAMAGE_556 * PRACTICE_MULTIPLIER
+	armor_penetration = ARMOR_PENETRATION_556 * PRACTICE_MULTIPLIER
+	embed = FALSE
+	sharp = FALSE
+	can_ricochet = FALSE
 
-/obj/item/projectile/bullet/c45r
-	name = "rubber bullet"
-	damage = 5
-	agony = 40
-	embed = 0
-	sharp = 0
-
-/obj/item/projectile/bullet/a50
-	damage = damage_50
+/obj/item/projectile/bullet/c65
+	damage = DAMAGE_65
+	armor_penetration = ARMOR_PENETRATION_65
+	penetrating = 1
+	sharp = FALSE
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/a50r
+/obj/item/projectile/bullet/c65/rubber
 	name = "rubber bullet"
-	damage = 10
-	agony = 75
-	embed = 0
-	sharp = 0
+	damage = DAMAGE_65 * RUBBER_DAMAGE_MULTIPLIER
+	agony = DAMAGE_65 * RUBBER_AGONY_MULTIPLIER
+	armor_penetration = ARMOR_PENETRATION_65 * RUBBER_PENETRATION_MULTIPLIER
+	embed = FALSE
+	sharp = FALSE
+	can_ricochet = FALSE
 
 /obj/item/projectile/bullet/a762
-	damage = damage_762
-	armor_penetration = AP_762
+	damage = DAMAGE_762
+	armor_penetration = ARMOR_PENETRATION_762
 	penetrating = 1
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/hv/a762
-	damage = HV_MULT*damage_762
-	armor_penetration = HV_MULT*AP_762
+/obj/item/projectile/bullet/a762/high_velocity
+	damage = DAMAGE_762 * HIGH_VELOCITY_MULTIPLIER
+	armor_penetration = ARMOR_PENETRATION_762 * HIGH_VELOCITY_MULTIPLIER
 	penetrating = 2
+	step_delay = HIGH_VELOCITY_STEP_DELAY
+
+//Revolvers and high-caliber pistols
+/obj/item/projectile/bullet/a357
+	damage = DAMAGE_357
+	armor_penetration = ARMOR_PENETRATION_REVOLVER
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/a556
-	damage = damage_556
-	armor_penetration = AP_556
-	penetrating = 2
+/obj/item/projectile/bullet/a357/high_velocity
+	damage = DAMAGE_357 * HIGH_VELOCITY_MULTIPLIER
+	armor_penetration = ARMOR_PENETRATION_REVOLVER * HIGH_VELOCITY_MULTIPLIER
+	penetrating = 1
+	step_delay = HIGH_VELOCITY_STEP_DELAY
+
+/obj/item/projectile/bullet/c38
+	damage = DAMAGE_38
+	armor_penetration = ARMOR_PENETRATION_REVOLVER
 	can_ricochet = TRUE
 
-/obj/item/projectile/bullet/a556p
-	damage = 10
-	agony = 10
-	embed = 0
-	sharp = 0
+/obj/item/projectile/bullet/c38/rubber
+	name = "rubber bullet"
+	damage = DAMAGE_38 * RUBBER_DAMAGE_MULTIPLIER
+	agony = DAMAGE_38 * RUBBER_AGONY_MULTIPLIER
+	armor_penetration = ARMOR_PENETRATION_REVOLVER * RUBBER_PENETRATION_MULTIPLIER
+	embed = FALSE
+	sharp = FALSE
+	can_ricochet = FALSE
```

Comment:
I'm not seeing the logic in making rubber rounds not ricochet. Rubber is bouncy!

Reply:
Will remove ricochet vars so they are inherited from the parent, the new-found ability for rubber bullets to ricochet won't be listed in the change-log, however. If you'd be able to include it in Rosalimo's big changelog he commits, that'd be appreciated.